provider/local.go

```diff
@@ -2,46 +2,87 @@ package provider
 
 import (
 	"encoding/json"
+	"log"
 	"net/http"
-	"skybin/core"
+
+	"github.com/gorilla/mux"
 )
 
+type localServer struct {
+	provider *Provider
+	logger   *log.Logger
+	router   *mux.Router
+}
+
+func NewLocalServer(provider *Provider, logger *log.Logger) http.Handler {
+
+	router := mux.NewRouter()
+
+	server := localServer{
+		provider: provider,
+		logger:   logger,
+		router:   router,
+	}
+
+	router.HandleFunc("/info", server.postInfo).Methods("POST")
```

Comment:
I might change the POST to PUT (client should give the full info object). 

We also still need a GET /info endpoint so we can see what the current configuration is.

Reply:
There is a GET /info endpoint on the public facing service that has all of the info that we would post to the metaserver. 

Do we need to set anything that we wouldn't be able to access publicly? 

If not, is it worth binding (what would essentially be the same function) to the local server?